kolibri/plugins/coach/assets/src/modules/examReport/handlers.js

```diff
@@ -16,11 +17,9 @@ export function showExamReportPage(store, params) {
         LearnerGroupResource.fetchCollection({ getParams: { parent: classId } }),
         ExamResource.fetchCollection({ getParams: { collection: classId }, force: true }),
         exam.question_sources.length
-          ? ContentNodeSlimResource.fetchCollection({
+          ? ContentNodeResource.fetchCollection({
```

Comment:
What fields are needed that are not present on ContentNodeSlimResource?

Reply:
don't recall offhand - will follow up